index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="src/index.scss"/>
```

Comment:
we cant use scss directly in the index.html
also it doesn't work for me. check if you have preloaded fonts in the network tab
should be like on screen
![Screenshot 2024-11-05 at 22 59 17](https://github.com/user-attachments/assets/7fccfff0-d3e6-4d5c-ac19-99f068480a67)